### PR TITLE
Bug 1861045 - set group duration to 0 if its value is not numeric

### PR DESCRIPTION
**Pull request by Archaeopteryx**

### CODE REVIEW

File: treeherder/log_parser/failureline.py [hunk 1]
```diff
@@ -132,7 +132,11 @@ def create_group_result(job_log, line):
         )
     else:
         group, _ = Group.objects.get_or_create(name=group_path[:255])
-        duration = int(line.get('duration', 0))
+        duration = line.get('duration', 0)
+        if type(duration) not in ['float', 'int']:
+            duration = 0
```
Comment (marco-c) [inline treeherder/log_parser/failureline.py:137]: Should we assume it's 0 when it is not numerical? Or should we use something like `null` to signify that the duration is not known?

Comment (Archaeopteryx) [reply treeherder/log_parser/failureline.py:137]: There is other fallback code directly below which sets it to `0`, and the new code follows this concept.

**Comment by jmaher** [comment]
please add a unittest for this

**Review by jmaher — APPROVED** [review]
thanks

**Pull request merged**